expression-language.md

```diff
@@ -0,0 +1,369 @@
+# CloudEvents SQL Expression Language
+
+## Abstract
+
+The goal of this specification is to define an expression language SQL-like which can be used to express predicates on
```

Comment:
at some point run it thru that md prettier tool so it wraps stuff at 80 columns

Reply:
This is based on an existing SQL RFC, right? If so, perhaps we should mention it and say whether it's compliant or where the differences are.

Reply:
I wrapped it at 120, is that ok? I don't want to have tables too much shrinked